app/src/main/java/echoserver/ConsoleIo.java

```diff
@@ -4,12 +4,12 @@
 import java.io.IOException;
 import java.io.InputStreamReader;
 
-public class Console {
+public class ConsoleIo {
   public static void print(String output) {
     System.out.println(output);
   }
 
-  public static void err(String message, IOException e) {
+  public static void err(String message, Exception e) {
```

Comment:
Why `Exception` vs `IOException`?

Reply:
Idk why but thanks for catching! Updated 😎